src/registrar/models/transition_domain.py

```diff
@@ -5,14 +5,18 @@
 
 class StatusChoices(models.TextChoices):
     READY = "ready", "Ready"
```

Comment:
Just realized this will make your other PR fail, this will definitely have to be kept in mind

Reply:
Good catch. In my latest updates I matched the transitionDomain model in both branches (along with the migration) so hopefully there will be no collisions there.